src/main/java/org/folio/list/services/export/ListExportService.java

```diff
@@ -132,12 +132,13 @@ private void doAsyncExport(ExportDetails exportDetails) {
       cancelExport,
       "Cancel export for list " + exportDetails.getList().getId()
     );
-
+    UUID userId = executionContext.getUserId();
+    log.info("YYZ Using user {} as proxy user for export", userId);
```

Comment:
YYZ?

Reply:
For easy log searching lol. Good catch, thought I removed those!